src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_31">
+      <div class="calendar__day calendar__day--start-day_sun"></div>
```

Comment:
You should implement start-day modifier for the calendar block.

Change modifiers names to a complex modifier.

Make sure, that you check your task according to the checklist before pull request.

![Screenshot 2022-07-22 at 10 36 28](https://user-images.githubusercontent.com/17098736/180388482-7958bafd-3b7d-48ef-accf-0cf1023f5cfa.png)

```suggestion
    <div class="calendar calendar--month-length--31 calendar__day--start-day--sun">
      <div class="calendar__day "></div>
```